pkg/storaged/storage.css

```diff
@@ -724,3 +664,35 @@ td.storage-action {
 .ct-form-box {
     margin-bottom: 1em;
 }
+
+/* Overview side panels */
+
+#storage .storage-sidebar .table {
+    table-layout: fixed;
+}
+
+.sidepanel-row {
+    display: flex;
+    flex-direction: row;
+}
+
+.sidepanel-row :nth-child(1) {
+    flex: 1;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+}
+
+.sidepanel-row :nth-child(2) {
+    align-self: center;
+    white-space: nowrap;
+}
+
+.sidepanel-row .spinner {
+    display: inline-block;
+}
+
+.sidepanel-row .rate {
+    width: 15c;
```

Comment:
This isn't a valid unit. Perhaps it's supposed to be `15ch`?

(This one is probably my fault, actually. I think I worked on this CSS. Well, oops. The intention is to not have the width change when the data # changes.)

Reply:
We don't need this anymore now that the rate is no longer displayed.